saleor/webhook/transport/asynchronous/transport.py

```diff
@@ -251,13 +317,76 @@ def trigger_webhooks_async(
         )
 
 
+def _generate_deferred_payload(
+    event_type: str,
+    webhook: "Webhook",
+    payload_args: dict,
+    delivery: "EventDelivery",
+    logger_obj: "logging.Logger",
+):
+    args_obj = DeferredPayloadData(**payload_args)
+    requestor = None
+    if args_obj.requestor_model_id and args_obj.requestor_model_name in (
+        RequestorModelName.APP,
+        RequestorModelName.USER,
+    ):
+        model = apps.get_model(args_obj.requestor_model_name)
+        requestor = model.objects.filter(pk=args_obj.requestor_model_id).first()
+
+    subscribable_object = (
+        apps.get_model(args_obj.model_name).objects.filter(pk=args_obj.model_id).first()
+    )
+    if not subscribable_object:
+        logger_obj.error(
```

Comment:
IMHO, this could be quite a common case. Like checkout-updated to deliver, but the order was created and checkout was deleted. Are you sure, that we want to raise this as an error?

Reply:
I'm not sure yet; I can change this to a warning as I think we should monitor situations when this happens first.

Reply:
I think this will be quite common case. Like `checkout_updated` events + checkout->order. This could raise this `error`. This will also be handled by active sentry. So could cause some false positives. I would consider changing this to the warning.